lib/src/main/java/com/evervault/Evervault.java

```diff
@@ -94,10 +102,24 @@ public Evervault(String apiKey, EcdhCurve ecdhCurve, String[] ignoreDomains) thr
     }
 
     public Evervault(String apiKey, EcdhCurve ecdhCurve, Boolean intercept, String[] ignoreDomains) throws EvervaultException {
+        this(apiKey, ecdhCurve, intercept, ignoreDomains, null);
+        System.out.println(
```

Comment:
Do you want to add the warning to the constructors where ignoreDomains (Lines 88 and 100) is added and intercept (Line  96) as any of these could be used?

Reply:
Ah sorry yes 👍